tests/unit/lib/test_ml_plugins.py

```diff
@@ -120,7 +120,6 @@ def test_disable_via_config_change(
 
         self.harness.update_config({"plugin_opensearch_knn": False})
         self.charm.plugin_manager.check_plugin_manager_ready.assert_called()
-        self.charm._restart_opensearch_event.emit.assert_called_once()
```

Comment:
This line will be changed anyways on [DPE-4196 PR](https://github.com/canonical/opensearch-operator/pull/435/files#diff-edbec226d3bd45d1b8bb2a562d8c71e006c439ea5bf75d0a5e9cf490c7a41f0eR136)